internal/drivers/util.go

```diff
@@ -0,0 +1,16 @@
+package drivers
+
+import (
+	"context"
+
+	"github.com/drone-runners/drone-runner-aws/types"
+)
+
+func isHosted(ctx context.Context) bool {
```

Comment:
move this to `harness/context` instead as it's only relevant to harness

Reply:
this is used by manager.go, putting it in harness/context will make it cyclic dependency.